examples/user_guide/12-Responding_to_Events.ipynb

```diff
@@ -363,16 +364,18 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "jane_dmap = hv.DynamicMap(lambda dollars: hv.Text(0.5,0.5,\"Jane's balance: $%s\" % dollars), \n",
-    "              streams=dict(dollars=janes_account.param.balance))\n",
-    "jane_dmap.opts(height=100, xaxis=None, yaxis=None)"
+    "bank_dmap = hv.DynamicMap(lambda total, overdraft: hv.Table({'balance':['$%s' % total],\n",
+    "                                                    'overdraft limit':['$%s' % overdraft]}, \n",
+    "                                                   ['balance', 'overdraft limit']), \n",
+    "              streams=dict(total=account.param.balance, overdraft=account.param.overdraft))\n",
+    "bank_dmap.opts(height=100, xaxis=None, yaxis=None, title=\"Jane's bank account\")"
```

Comment:
```python
account = BankAccount(name='Jane', balance=300)
```

For clarity I'd probably pull it out, because that looks really overwhelming:

and then:

```python
streams = dict(total=account.param.balance, overdraft=account.param.overdraft, owner=account.param.name)

def table(owner, total, overdraft):
    return hv.Table([(owner, overdraft, total)], ['Owner', 'Overdraft ($)', 'Total ($)'])

bank_dmap = hv.DynamicMap(table, streams=streams)

bank_dmap.opts(height=100)
```

Reply:
Yes, that is definitely more readable. Addressed in 4cdcdb8c7.